src/main/resources/templates/AdvisorRHEL/resolvedRecommendationInstantEmailBody.html

```diff
@@ -31,7 +31,7 @@
                                                     <tr>
                                                         <td class="rh-card__header">
                                                             <h2>
-                                                                New recommendations
+                                                                Resolved recommendations
```

Comment:
I see that this templates *replaces* the "New recommendations" email, we need both, the 'new' and 'resolved'

Reply:
It actually doesn't replace it. It's another file (AdvisorRHEL/resolvedRecommendationInstantEmailBody.html ) I copy pasted the files from new recommendations first, committed it then to let josejulio take a look at it and then renamed it afterwards.